src/TMS/dev/TMP117.cpp

```diff
@@ -25,4 +25,6 @@ uint16_t TMP117::readTemp() {
     return temp;
 }
 
+TMP117::TMP117() {}
```

Comment:
Move this up to the top of the file. It doesn't really matter, but I generally try to keep the methods in the same order as they are in the hpp file